src/NewsService.Mappers/Models/DbNewsMapper.cs

```diff
@@ -24,8 +23,9 @@ public DbNews Map(News request)
                 AuthorId = request.AuthorId,
                 SenderId = request.SenderId,
                 DepartmentId = request.DepartmentId,
-                CreatedAt = DateTime.UtcNow,
-                IsActive = true
+                IsActive = true,
+                CreatedBy = Guid.NewGuid(),
```

Comment:
what is this?

Reply:
Id news, so that when editing news, CreatedBy does not change, but ModifedBy is added

Reply:
CreatedBy is a property that contains userId who created the news. You must use _httpContextAccessor.HttpContext.GetUserId() to get sender userId of the news. Now you save random Guid, a user with the random Guid does not exist.

Reply:
done